src/dtos/ProductCreateDto.ts

```diff
@@ -0,0 +1,5 @@
+export interface ProductCreateDto {
+  name?: string;
+  price: number;
+  description?: string;
+}
```

Comment:
```
    Answer.King.Api.RequestModels.ProductDto:
      type: object
      properties:
        name:
          type: string
          nullable: true
        description:
          type: string
          nullable: true
        price:
          type: number
          format: double
        category:
          $ref: '#/components/schemas/Answer.King.Api.RequestModels.CategoryId'
      additionalProperties: false
```

seems like the api spec also wants the category for creating products

Reply:
Don't think the create product model is really necessary tbh